index.js

```diff
@@ -1,7 +1,15 @@
 'use strict';
 const Build = require('./lib/build');
+const factory = require('./lib/factory');
 const Job = require('./lib/job');
 const Pipeline = require('./lib/pipeline');
 const User = require('./lib/user');
 
-module.exports = { Build, Job, Pipeline, User };
+module.exports = {
+    Build,
+    configureBuildModel: factory.configureBuildModel,
```

Comment:
> should every model just have a factoryMethod that will retrieve the instance of a model? 

Are you asking if there will be a method for each method, like `getJobModel`, `getPipelineModel`?